packages/xo-web/src/common/intl/messages.js

```diff
@@ -463,7 +463,8 @@ const messages = {
   migrateBackupSchedule: 'Migrate to the new backup system',
   migrateBackupScheduleMessage:
     'This will convert the legacy backup job to the new backup system. This operation is not reversible. Do you want to continue?',
-  runBackupNgJobConfirm: 'Are you sure you want to run {name} ({id})?',
+  runBackupNgJobConfirm:
+    'The job {name} ({id}) will backup {nVms, number} VM{nVms, plural, one {} other {s}}. Are you sure you want to run it?',
```

Comment:
So what about [this](https://github.com/vatesfr/xen-orchestra/pull/4875#discussion_r396994756)? The meaning has changed, right? I think we could keep the original message as it was and create a new one for the number of VMs. This would allow us to keep the existing translations and still give useful information about the number of VMs in any language. What do you think?